sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/ShareDirectoryStorageResourceContainer.cs

```diff
@@ -22,12 +24,23 @@ internal class ShareDirectoryStorageResourceContainer : StorageResourceContainer
 
         public override string ProviderId => "share";
 
+        internal StorageResourceItemProperties ResourceProperties;
```

Comment:
Is this actually used anywhere? A container object should not need a `StorageResourceItemProperties` afaik and its also not on the base `StorageResourceContainer`. Perhaps I missed it somewhere but if it's not used, we should just remove this and the constructor for it. In `PathScanner` we just won't pass anything for a directory.

Reply:
Ah good catch, I was using it at some point, will remove it now.